### PR TITLE
Added Deepsource - A company that pays Technical writers

### PR DESCRIPTION
**Pull request by PhilzAce1**
Deepsource pays $150 
Respond within a day 
and Pays Via paypal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Dev Spotlight](https://www.devspotlight.com/jobs/) - Around $300-$500 per piece depending on length and content
   > Technical content production agency that works with many clients.
 
+- [Deepsource](https://deepsource.io/tech-writer/) - Around $150 per piece  
+  > Technical Content concerning code quality, code review and static analysis
```
Comment (karllhughes) [inline README.md:35]: Alphabetically, this should be above Dev Spotlight.

**Comment by malgamves** [comment]
Hey @PhilzAce1 could you change the order of the entry?

**Comment by saif-deepsource** [comment]
hey @PhilzAce1 thanks for opening this PR. I take care of the [DeepSource tech writer program](https://deepsource.io/tech-writer/). 😀
Can you fix the following things-
1. change the order of the entry
2. there will capital "S" in DeepSource

**Comment by malgamves** [comment]
Looks great! Thanks @saif-deepsource and @PhilzAce1

**Pull request merged**